packages/widget/src/components/Header/SettingsButon.style.tsx

```diff
@@ -0,0 +1,32 @@
+import { darken, styled } from '@mui/material/styles';
+import { IconButton } from '@mui/material';
+import { getInfoBackgroundColor, getWarningBackgroundColor } from '../../utils';
+
+interface SettingsIconButtonProps {
+  notification?: 'info' | 'warning';
+}
+
+export const SettingsIconButton = styled(IconButton, {
+  shouldForwardProp: (props) => props !== 'notification',
```

Comment:
I figured this out - not having this doesn't cause any console warnings but if the prop you are passing is a string that string will be added to the DOM. For example

without shouldForwardProp

![without-shouldForwardProp](https://github.com/lifinance/widget/assets/7836726/593f6f0e-34a4-449b-9a52-6c8a40ece3a1)

and with shouldForwardProp

![withShouldForwardProps-button](https://github.com/lifinance/widget/assets/7836726/ed04a437-91da-400a-b112-2e29bf08a4d7)